consensus/votecounter/ballot.go

```diff
@@ -0,0 +1,41 @@
+package votecounter
+
+import "github.com/NethermindEth/juno/consensus/types"
+
+type VoteType uint8
+
+const (
+	Prevote VoteType = iota
+	Precommit
+)
+
+type ballot []bool
```

Comment:
Are ballots a dynamic slice, or can their size be known before hand. Looking down the initialized with size two, so maybe they can by `type ballot [2]bool`?